packages/edit-site/src/components/page-pages/index.js

```diff
@@ -136,7 +139,8 @@ export default function PagePages() {
 				header: 'Status',
 				id: 'status',
 				cell: ( props ) =>
-					postStatuses[ props.row.original.status ]?.name,
+					postStatuses[ props.row.original.status ] ??
+					props.row.original.status,
```

Comment:
Where does this come from?

Reply:
oh you show the key. Fine 👍

Reply:
Do we need to show the key here?

Reply:
There's an advantage to pre-filling the table with the slug status over starting with an empty string: it acts as a better placeholder for the content.

By using the slug as fallback, the table already has the size of the content (because the slug for `Publish` is `publish` and for `Draft` is `draft`, etc.). This avoids "_visual resizes of the table_" later to adapt the column to the new content. I've uploaded some before/after videos to the description. While it's not the same issue, it can serve as an example of the visual effect resizing has when the status starts as an empty string in `trunk`.